src/SSVOpenHexagon/Core/HGGraphics.cpp

```diff
@@ -273,6 +327,10 @@ void HexagonGame::draw()
     render(pivotQuads, getRenderStates(RenderStage::PivotQuads));
     render(playerTris, getRenderStates(RenderStage::PlayerTris));
 
+    render(wallQuads3DTop, getRenderStates(RenderStage::WallQuads3D));
+    render(pivotQuads3DTop, getRenderStates(RenderStage::PivotQuads3D));
+    render(playerTris3DTop, getRenderStates(RenderStage::PlayerTris3D));
```

Comment:
Can this also be wrapped in `if (renderAbove) { ... }`?

Reply:
yes